.env

```
@@ -27,6 +27,7 @@ NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_NAME=no-aws-header-name
 NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_VALUE=no-aws-header-value
 NEXT_PUBLIC_VILNACRM_GMAIL=info@vilnacrm.com
 NEXT_PUBLIC_VILNACRM_PRIVACY_POLICY_URL=https://github.com/VilnaCRM-Org/
+NEXT_PUBLIC_PROD_API_URL=https://api.vilnacrm.com
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_PROD_API_URL key should go before the NEXT_PUBLIC_VILNACRM_GMAIL key

(UnorderedKey)